bot/bot_secrets.py

```diff
@@ -108,6 +109,18 @@ def bot_prefix(self, value: str) -> None:
         if self._botPrefix:
             raise ConfigAccessError(f'bot_prefix has already been initialized')
         self._botPrefix = value
+    
+    @property
+    def gif_me(self) -> str:
```

Comment:
we should postfix this value with token so the json and the property should be gifMeToken

Reply:
This would be in the bot secrets template?

Reply:
ya, and in the BotSecrets class, just add Token to the end of everything